CHANGES.md

```
@@ -1,133 +1,16 @@
-# Version 1.9.0
+# Version 1.8.1
 
-## What's Changed
-
-### Notable changes
+### Changes
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)